### DISCUSSION TITLE
Zone numbers < 10 should have a leading 0 in their name

### DISCUSSION THREAD

**Discussion by Tipalo** [ufodone/envisalink_new#81] [General]
Greetings,  Nice new Integration good to see.

Having more than 10 zones places the zones out of order.  Having 64 zones was allot to go through to correct.  Even when zones were renamed with leading 0's they still were sorted out of order.

Eg:  58, 59,6,60,... 64,7,8,9

**Comment by chelming** [comment]
where are you using this sorting? when using the auto entities card I just add a sort for the zone attribute and set number: true and it works 1..2..10..20

context: I have 21 zones
![Screenshot_20240228-082431.png](https://github.com/ufodone/envisalink_new/assets/7746625/dbbb4e1c-e11f-420e-8a99-a4021c1b0d33)
